### PR TITLE
Code of conduct

### PR DESCRIPTION
**Pull request by Zerocreative0815**
added the code of conduct as as txt file to the projet

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.idea
```
Comment (nipafx) [inline .gitignore:1]: This file should not be part of this pull request.

:bulb: Before creating a new branch to work on, make sure you're on the most recent version of `master`.

File: README.md [hunk 2]
```diff
@@ -4,3 +4,5 @@
 #Bene, Leon, Peter and Janos
 
 code, comments and conversation will be in english
+
+#We chose Contributor Covenant as our Code of Conduct
```
Comment (nipafx) [inline README.md:8]: A CoC is important but maybe not so important to get a level-1 header. ;) More common would be something like:

```
## Code of Conduct

This project adopted the [Contributor Covenant code of conduct](path/to/file/in/repo).
```

**Review by nipafx — CHANGES_REQUESTED** [review]
A few changes:

**Comment by nipafx** [comment]
Re filename, see [my answer here](https://github.com/Cryxnoob/TeamProjectTH-BingenEP-1/pull/8#issuecomment-319464213).

File: .gitignore (hunk 1 above)
Comment (Zerocreative0815) [reply .gitignore:1]: so its only the *.iml and the out/* ?

is there a problem? because i was on the most recent version i think...

File: README.md (hunk 2 above)
Comment (Zerocreative0815) [reply README.md:8]: ok :D

File: .gitignore (hunk 1 above)
Comment (nipafx) [reply .gitignore:1]: I mean that the entire `.gitignore` file does not belong into this pull request. Let's discuss [here](https://github.com/Cryxnoob/TeamProjectTH-BingenEP-1/pull/14#issuecomment-319466219).
